src/core/Akka.Tests/Actor/TimerStartupCrashBugFixSpec.cs

```diff
@@ -0,0 +1,97 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="TimerStartupCrashBugFixSpec.cs" company="Akka.NET Project">
+//      Copyright (C) 2009-2024 Lightbend Inc. <http://www.lightbend.com>
+//      Copyright (C) 2013-2024 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+#nullable enable
```

Comment:
Enabled `nullable` for just this test - on my quest to ensure that we start doing that generally everywhere.